packages/bull-vault/test/e2e/BullStrategyTestFork.t.sol

```diff
@@ -96,6 +96,32 @@ contract BullStrategyTestFork is Test {
         assertEq(IERC20(usdc).balanceOf(user1), usdcToBorrow);
     }
 
+    function testSecondDeposit() public {
+        uint256 crabToDepositInitially = 10e18;
+
+        vm.startPrank(user1);
+        (uint256 wethToLend, uint256 usdcToBorrow) = _deposit(crabToDepositInitially);
+        vm.stopPrank();
+
+        assertEq(bullStrategy.balanceOf(user1), crabToDepositInitially);
+        assertEq(IEulerDToken(dToken).balanceOf(address(bullStrategy)), usdcToBorrow);
+        assertTrue(wethToLend.sub(IEulerEToken(eToken).balanceOfUnderlying(address(bullStrategy))) <= 1);
+        assertEq(IERC20(usdc).balanceOf(user1), usdcToBorrow);
+
+        uint256 userUsdcBalanceBefore = IERC20(usdc).balanceOf(user1);
+        uint256 userBullBalanceBefore = bullStrategy.balanceOf(user1);
+        uint256 crabToDepositSecond = 7e18;
+        uint256 bullToMint = _calcBullToMint(crabToDepositSecond);
+        vm.startPrank(user1);
+        (uint256 wethToLendSecond, uint256 usdcToBorrowSecond) = _deposit(crabToDepositSecond);
+        vm.stopPrank();
+
+        assertEq(bullStrategy.balanceOf(user1).sub(userBullBalanceBefore), bullToMint);
+        assertEq(IEulerDToken(dToken).balanceOf(address(bullStrategy)).sub(usdcToBorrow), usdcToBorrowSecond);
+        assertTrue(wethToLendSecond.sub(IEulerEToken(eToken).balanceOfUnderlying(address(bullStrategy)).sub(wethToLend)) <= 1);
```

Comment:
This is fine for this test, but if you add a fuzzing one, note that sometimes it gets to 2 wei difference.

Reply:
Added issue https://github.com/opynfinance/squeeth-monorepo/issues/696